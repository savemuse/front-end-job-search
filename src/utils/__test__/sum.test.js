import sum from '../sum';

describe('sum', () => {
  it('should be add two numbers', () => {
    expect(sum(1, 2)).toBe(3);
  });

  it('should be match snapshot', () => {
    expect(sum(1, 2)).toMatchSnapshot();
  });
});
