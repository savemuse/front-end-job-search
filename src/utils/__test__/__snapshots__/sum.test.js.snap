// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`sum should be match snapshot 1`] = `3`;
